WebApplication/Backend/Repositorys/PatientRepository.cs

```diff
@@ -30,26 +28,25 @@ public PatientRepository()
         ///<exception>
         ///if any exception occurs method will return null
         ///</exception>
-        internal List<Patient> GetAllPatients()
+        internal List<Patient> GetPatients(String sqlDml)
```

Comment:
You shouldn't set the sql query through parameter, this is repository's job, not service's